requirements.txt

```diff
@@ -0,0 +1,4 @@
+git+https://github.com/edx/XBlock.git#egg=XBlock==0.4.12
+git+https://github.com/edx/xblock-utils.git#egg=xblock-utils==1.0.2
+django==1.8.12
```

Comment:
actually, is there a reason we don't want to support earlier versions of Open edX, at least one back to Cypress?